packages/contracts/test/core/permission/permission-manager.ts

```diff
@@ -20,10 +20,10 @@ const RESTRICTED_PERMISSIONS_FOR_ANY_ADDR = [
   ethers.utils.id('TEST_PERMISSION_2'),
 ];
 
-const UNSET_FLAG = ethers.utils.getAddress(
+export const UNSET_FLAG = ethers.utils.getAddress(
   '0x0000000000000000000000000000000000000000'
 );
-const ALLOW_FLAG = ethers.utils.getAddress(
+export const ALLOW_FLAG = ethers.utils.getAddress(
   '0x0000000000000000000000000000000000000002'
 );
 export const ANY_ADDR = '0xffffffffffffffffffffffffffffffffffffffff';
```

Comment:
shouldn't these be in a global contast place?

Reply:
We don't have a global constants file for OSX as subgraph has. All the constants are scattered everywhere. 
Maybe this needs some general refactoring at some point, which might also be important for the code split.

Here, the export was even redundant so I removed it in e4f3fc28fc0b4e2d266863566e919ec6d8ec3741.